src/mergeStandard.ts

```diff
@@ -0,0 +1,24 @@
+import { StandardType } from './standard';
+
+type ChangeType = 'breaking' | 'non-breaking' | 'unclassified';
```

Comment:
have a look at those hardcodes, also you have these same variables in `standard.ts`. I guess that for now with many PRs there is no other way to solve it but please remember to clean it up after all is merged, refactor to have these declared in one place

Reply:
Yup, we are going to move all these types in a separate file.